airbyte-integrations/connectors/source-facebook-marketing/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM airbyte/source-facebook-marketing:latest
+
+COPY . ./airbyte/integration_code
+RUN pip install ./airbyte/integration_code
+
+COPY source_facebook_marketing ./source_facebook_marketing
+
+# The entrypoint and default env vars are already set in the base image
+ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
+ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
```

Comment:
I think that we don't need to commit this file, since they have removed it from the repo, right?

Reply:
Yes! I left it just so we could replicate this, but it's not necessary :raised_hands: I'll remove it for final review :smile: